addons/stock/report/report_stock_forecasted.py

```diff
@@ -220,7 +220,7 @@ def _reconcile_out_with_ins(lines, out, ins, demand, product_rounding, only_matc
                 if float_is_zero(demand, precision_rounding=product_rounding):
                     continue
                 current = currents[product.id]
-                taken_from_stock = min(demand, current)
+                taken_from_stock = min(demand, current) if out.procure_method != 'make_to_order' else 0
```

Comment:
I think It's not exactly true. If you have a stock rule in `mts_else_mto`, you will have a stock move with `procure_method == 'make_to_order'`. 
Depending on the quantity in stock we will either take it if the move quantity < stock quantity. Either replenish in mto if the move quantity > stock quantity